### PR TITLE
Add tests for bundles package

### PR DESCRIPTION
**Pull request by ashishmax31**

### CODE REVIEW

**Comment by ashishmax31** [comment]
For now I've pushed the registry image to my quay repo. I've created a PR in app-interface to mirror the `registry` repo from dockerhub to the app-sre quay repo but, it needs an approval from Amador. (Once its merged, I'll change change the image url).

**Review by yashvardhan-kukreja — APPROVED** [review]
lgtm!

**Pull request merged**
